layers/+lang/mercury-layer/README.org

```diff
@@ -0,0 +1,37 @@
+#+TITLE: mercury layer
+
+# The maximum height of the logo should be 200 pixels.
```

Comment:
Please remove this line its just for introduction into the layer and should not be left in the finished layer.